setup.cfg

```diff
@@ -0,0 +1,84 @@
+##########################
+# Setup.py Configuration #
+##########################
+
+[metadata]
+name = yaml2sbml
+version = attr: yaml2sbml.version.__version__
+description = A small package to convert ODEs specified in a YAML file to SBML/PEtab
+long_description = file: README.md
+long_description_content_type = text/markdown
+
+# URLs
+url = https://github.com/yaml2sbml-dev/yaml2sbml
+download_url = https://github.com/yaml2sbml-dev/yaml2sbml/releases
+project_urls =
+    Bug Tracker = https://github.com/yaml2sbml-dev/yaml2sbml/issues
+    Documentation = https://yaml2sbml.readthedocs.io
+    Changelog = https://yaml2sbml.readthedocs.io/en/latest/release_notes.html
+
+# Author information
+author = Jakob Vanhoefer, Marta R. A. Matos, Dilan Pathirana, Yannik Schälte
+author_email = jakob.vanhoefer@uni-bonn.de,marta.ra.matos@gmail.com
+
+# License information
+license = MIT
+license_file = LICENSE
+
+# Search tags
+classifiers =
+    Development Status :: 4 - Beta
+    Environment :: Console
+    Intended Audience :: Science/Research
+    License :: OSI Approved :: MIT License
+    Operating System :: OS Independent
+    Programming Language :: Python
+    Programming Language :: Python :: 3.7
+    Programming Language :: Python :: 3.8
+    Programming Language :: Python :: 3.9
+keywords = YAML, SBML, PEtab, ODE
+
+[options]
+install_requires =
+    numpy >= 1.19.4
+    scipy >= 1.6.0
```

Comment:
scipy is actually _not_ used in the public API, but just in the examples...

Reply:
good to know, my mistake! will move it back to the examples.

Reply:
Now it should be exactly as before, also again including python 3.6.